tests/data/empty_machineconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Super nit picky here, but we could elaborate on the name. To me, an empty machine config is `items: []`. But we could name it `nil_machineconfig_list.json`.

Also, this is certainly something I can bike shed later in a separate PR.